template/runtime/src/lib.rs

```diff
@@ -219,6 +219,7 @@ impl pallet_grandpa::Config for Runtime {
 	type HandleEquivocation = ();
 
 	type WeightInfo = ();
+	type MaxAuthorities = MaxAuthorities;
```

Comment:
I accidentally updated the Substrate dependency all the way to the tip of Substrate master. But I believe this is the only change that is not about scale-info, so I kind of prefer to just keep it.